jbmc/src/java_bytecode/java_object_factory.cpp

```diff
@@ -496,6 +496,7 @@ static mp_integer max_value(const typet &type)
   else if(type.id() == ID_unsignedbv)
     return to_unsignedbv_type(type).largest();
   UNREACHABLE;
+  return 0;
```

Comment:
It seems like this change (and the others in this commit) are somewhat independent of the stated purpose of this  PR or are these warnings triggered by the change in switch/case statements?

Reply:
Thanks for the clarification - I'm fine with this PR as it is :-)